awx/main/utils/filters.py

```diff
@@ -10,6 +10,8 @@
 )
 import logging
 from logging import Filter
+from dateutil.tz import tzutc
+from datetime import datetime
```

Comment:
Probably these needs to be removed? The api-lint check is failing because they does not seem to be used https://github.com/ansible/awx/actions/runs/9116813905/job/25066241902?pr=15199#step:4:30

```suggestion
```